README.md

```diff
@@ -69,6 +69,14 @@ Here is an example that sets `NODE_VERSION` and `NODE_ENV` before running a buil
 export NODE_VERSION=8 NODE_ENV=production ; build npm run build
 ```
 
+### Testing it for a mono repository
+
+If you like to run a debugging build for your mono repository you need to pass the **Base directory** like in the Netlify UI.
+
+```
+./test-tools/test-build.sh path/to/site/repo base/directory 'your build command'
```

Comment:
@kitop added a description on how to run it for monorepos so I think your comment here should be resolved with it as you can test it now even correctly for monorepos locally (which was not possible before)

https://github.com/netlify/pillar-workflow/issues/895#issuecomment-1286850460